src/dawn/engine_dawn.cc

```diff
@@ -502,18 +520,41 @@ Result EngineDawn::CreateFramebufferIfNeeded(
   if (!result.IsSuccess())
     return result;
 
-  // First make the Dawn color attachment textures that the render pipeline will
-  // write into.
   {
     ::dawn::Texture fb_texture;
 
-    result =
-        MakeFramebufferTexture(*device_, fb_format, width, height, &fb_texture);
+    result = MakeTexture(*device_, fb_format, width, height, &fb_texture);
     if (!result.IsSuccess())
       return result;
     render_pipeline->fb_texture = std::move(fb_texture);
   }
 
+  // Then make the depthStencil attachment textures that the render pipeline
+  // will write into.
+  ::dawn::TextureFormat depthStencil_format{};
+  auto depthBuffer = render_pipeline->pipeline->GetDepthBuffer().buffer;
```

Comment:
I would recommend auto* here so we are sure this isn't making a copy of a big object.

Reply:
yes, good point!